operators/src/processing/meteosat/temperature.rs

```diff
@@ -0,0 +1,637 @@
+use crate::engine::{
+    ExecutionContext, InitializedRasterOperator, Operator, QueryContext, QueryProcessor,
+    RasterOperator, RasterQueryProcessor, RasterQueryRectangle, RasterResultDescriptor,
+    SingleRasterSource, TypedRasterQueryProcessor,
+};
+use crate::util::Result;
+use async_trait::async_trait;
+use TypedRasterQueryProcessor::F32 as QueryProcessorOut;
+
+use crate::error::Error;
+use futures::stream::BoxStream;
+use futures::StreamExt;
+use geoengine_datatypes::primitives::{Measurement, SpatialPartition2D};
+use geoengine_datatypes::raster::{
+    EmptyGrid, Grid2D, GridOrEmpty, GridShapeAccess, GridSize, NoDataValue, Pixel, RasterDataType,
+    RasterPropertiesKey, RasterTile2D,
+};
+use serde::{Deserialize, Serialize};
+
+// Output type is always f32
+type PixelOut = f32;
+use crate::processing::meteosat::satellite::{Channel, Satellite};
+use RasterDataType::F32 as RasterOut;
+
+const OUT_NO_DATA_VALUE: PixelOut = PixelOut::NAN;
+
+/// Parameters for the `Temperature` operator.
+/// * `force_satellite` forces the use of the satellite with the given name.
+#[derive(Debug, Serialize, Deserialize, PartialEq, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct TemperatureParams {
+    force_satellite: String,
+}
+
+impl Default for TemperatureParams {
+    fn default() -> Self {
+        TemperatureParams {
+            force_satellite: "".into(),
```

Comment:
we should use a number (option)

Reply:
See above: 64b4dde20c007702dee325a438a5337d7c88f069